src/lib.rs

```diff
@@ -194,14 +194,25 @@ for i in 1..10 {
 		.then!{ w += i * i }
 		.then_else!{ w += 1 };
 }
-assert_eq!(w, 181);
+assert_eq!(w, 72);
 # }
 ```
 */
 #[macro_export]
 macro_rules! then_else {
-	($v:tt, $($body:tt)*) => {
-		$v {
+	({ then! ( $cond:expr, $($if_body:tt)* ) }, $($body:tt)*) => {
```

Comment:
Could you do `$crate::then` here? It's not always certain the `then` macro is in scope.